recipes/gdal/post_3.5.0/conanfile.py

```diff
@@ -0,0 +1,496 @@
+    }, {
+        "dep": "SFCGAL",
+        "descr": "gdal core supports ISO 19107:2013 and OGC Simple Features" \
+                + " Access 1.2 for 3D operations"
+    }, {
+        "dep": "SPATIALITE",
+        "descr": "Enable spatialite support for sqlite3"
+    }, {
+        "conan_dep": "sqlite3::sqlite",
+        "require": "sqlite3/3.38.5",
+        "default": True,
+        "dep": "SQLite3",
+        "descr": "Enable SQLite3 support (used by SQLite/Spatialite, GPKG, " \
+                + "Rasterlite, MBTiles, etc.)"
+    }, {
+        "dep": "SWIG",
+        "descr": "Enable language bindings"
+    }, {
+        "dep": "Shapelib",
+        "descr": "Enable Shapelib support (not recommended, internal Shapelib" \
+                + " is preferred)."
+    }, {
+        "dep": "TEIGHA",
+        "descr": "Enable DWG and DGNv8 drivers"
+    }, {
+        "require": "libtiff/4.3.0",
+        "dep": "TIFF",
+        "default": True,
+        "descr": "Support for the Tag Image File Format (TIFF)."
+    }, {
+        "dep": "TileDB",
+        "descr": "enable TileDB driver"
+    }, {
+        "dep": "WebP",
+        "descr": "WebP compression"
+    }, {
+        "dep": "XercesC",
+        "descr": "Read and write XML formats (needed for GMLAS and ILI drivers)"
+    }, {
+        "require": "zlib/1.2.12",
+        "default": True,
+        "dep": "ZLIB",
+        "descr": "zlib (external)"
+    }, {
+        "conan_dep": "zstd::zstdlib",
+        "require": "zstd/1.5.2",
+        "dep": "ZSTD",
+        "descr": "ZSTD compression library"
+    }, {
+        "dep": "rdb",
+        "descr": "enable RIEGL RDB library"
+    },
+    {
+        "conan_dep": [ "libkml::kmldom", "libkml::kmlengine" ],
+        "dep": "LibKML",
+        "require": "libkml/1.3.0"
+    }]
+
+    options = _make_options(gdal_deps)
+    default_options = _make_default_options(gdal_deps)
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build"
+
+    def export_sources(self):
+        self.copy("CMakeLists.txt")
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            self.copy(patch["patch_file"])
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.with_sqlite3:
+            self.options["sqlite3"].enable_column_metadata = True
+
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+
+        for dep in self.gdal_deps:
+            if "require" in dep and getattr(self.options, _option_name(dep)):
+                self.requires(dep["require"])
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version],
+                destination=self._source_subfolder, strip_root=True)
+
+    def _configure_cmake(self):
```

Comment:
You are missing our caching trick https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md#caching-helper

Reply:
added, thanks for the tip